src/site/xdoc/server/feature-performance.xml

```diff
@@ -78,26 +78,6 @@
       <p>More details on metrics are available on <a href="metrics.html">the metric page</a>.</p>
     </subsection>
 
-    <subsection name="Measuring dequeueing speed">
```

Comment:
I'd prefer we keep this for now, and check that is it still right as soon as possible.